fuzz/fuzz.go

```diff
@@ -0,0 +1,17 @@
+package fuzz
```

Comment:
Please move this to `internal/fuzz/fuzz.go` so that this doesn't become public API surface area.